es-bs-base/scss/utilities/_spacing.scss

```diff
@@ -7,6 +7,27 @@
     $infix: breakpoint-infix($breakpoint, $grid-breakpoints);
 
     @each $prop, $abbrev in (margin: m, padding: p) {
+      // print out the deprecated spacing classes first so conflicts can be overridden
```

Comment:
What conflicts are possible here?

Reply:
None actually - this comment was a vestige of the first naming scheme I had proposed, good catch. Removed.